api/api_test.go

```diff
@@ -99,9 +111,44 @@ func TestUtils(t *testing.T) {
 	tester := iffy.NewTester(t, hdl)
 
 	tester.AddCall("testMetrics", http.MethodGet, "/metrics", "").
+		Checkers(
+			iffy.ExpectStatus(200),
+		)
+
+	tester.AddCall("testPing", http.MethodGet, "/unsecured/mon/ping", "").
+		Checkers(
+			iffy.ExpectStatus(200),
+		)
+
+	tester.AddCall("testRootHandler", http.MethodGet, "/", "").
+		Headers(regularHeaders).
+		Checkers(
+			iffy.ExpectStatus(301),
+		)
+
+	tester.AddCall("testMeta", http.MethodGet, "/meta", "").
+		Headers(regularHeaders).
+		Checkers(
+			iffy.ExpectStatus(200),
+		)
+
+	utask.FMaintenanceMode = true
```

Comment:
be careful, this is misleading: the calls are declared with `tester.AddCall`, but they are only performed on `tester.Run()`
this means that `FMaintenanceMode` is effectively `true` for every call in this test
I would place this declaration either at the beginning of the test, with a deferred set to false
or right before `tester.Run()`, whatever seems more readable to you

Reply:
Fixed!
Thanks